Packs/CyberArkIdentity/README.md

```diff
@@ -0,0 +1,37 @@
+This pack includes XSIAM content.
+
```

Comment:
why do we need this sentence?

Reply:
Its a part of the pack README convention [here](https://docs.google.com/document/d/16CKdwnQF9r0IZPSZUM34NWFJHX1IEcdAe0vnvIrqZ2w/edit)

@tomer-pan @meital-aizen @tkatzir FYI

Reply:
It make sense when the pack it deployed to MP v1 as well, but this is not the case here.